File: fizzbuzz.go
package main

import "fmt"

func main() {
	for i := 0; i <= 100; i++ {
		if i%15 == 0 {
			fmt.Printf("FizzBuzz\n")
		}
		if i%3 == 0 {
			fmt.Printf("Fizz\n")
		}
		if i%5 == 0 {
			fmt.Printf("Buzz\n")
		}
		if i != 0 {
			fmt.Printf("%d", i)
		}
		fmt.Printf("\n")
	}
}

// Run
// go run fizzbuzz.go

File: fizzbuzz.js
// FizzBuzz = Sisa bagi 3 dan 5
// Fizz = Sisa bagi 3
// Buzz = Sisa bagi 5

for (i = 0; i <= 100; i++) {
    if (i % 15 == 0) {
        console.log("FizzBuzz")
    } else if (i % 3 == 0) {
        console.log("Fizz")
    } else if (i % 5 == 0) {
        console.log("Buzz")
    } else {
        console.log(i)
    }
}

// Run
// node fizzbuzz.js